internal/cli/command/cluster/import.go

```diff
@@ -0,0 +1,183 @@
+// Copyright © 2019 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cluster
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	pkgPipeline "github.com/banzaicloud/banzai-cli/.gen/pipeline"
+	"github.com/banzaicloud/banzai-cli/internal/cli"
+	"github.com/banzaicloud/banzai-cli/internal/cli/utils"
+	"github.com/goph/emperror"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"gopkg.in/AlecAivazis/survey.v1"
+)
+
+type importOptions struct {
+	name       string
+	file       string
+	kubeconfig string
+}
+
+// NewInstallCommand returns a cobra command for `install` subcommands.
+func NewImportCommand(banzaiCli cli.Cli) *cobra.Command {
+	options := importOptions{}
+
+	cmd := &cobra.Command{
+		Use:     "import",
+		Aliases: []string{"imp"},
+		Short:   "Manage cluster imports",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return importCluster(banzaiCli, options)
+		},
+	}
+
+	flags := cmd.Flags()
+	flags.StringVarP(&options.name, "name", "n", "", "Name of the cluster")
+	flags.StringVarP(&options.file, "file", "f", "", "Cluster descriptor file")
+
+	return cmd
+}
+
+func importCluster(banzaiCli cli.Cli, options importOptions) error {
+	pipeline := banzaiCli.Client()
```

Comment:
```suggestion
	client := banzaiCli.Client()
```

Then you don't have to alias the client package to `pkgPipeline`.